firebase-sessions/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,10 @@
   <!--<uses-sdk android:minSdkVersion="16"/>-->
 
   <application>
+    <service
+      android:name=".SessionDataService"
```

Comment:
Will this `.` carry on to the app when the manifests get merged? Will this have a different name for every app depending on it's app id? Or will it always be `...firebase-sessions.SessionDataService` ?

Reply:
I still see it show up as `com.google.firebase.sessions.SessionDataService` in stack traces when running in an app, but I'll make this fully qualified like the other one.